const handleError=(err,req,res,next)=>{
    console.log("error here")
}
module.exports={
    handleError
}